### ISSUE TITLE
Android User Authentication

### ISSUE THREAD

**Issue by CrazyChaoz** [tink-crypto/tink-java#12]
Your docs state 
> Android Keystore is only useful when you want to require user authentication for key use 

but your implementation looks like it would need a complete rewrite of the AndroidKeystoreKmsClient.java to be able to handle _user authentication for key use_.

Is that intended? Or is there an easy way around the limitations of the existing implementation?

**Comment by juergw** [comment]
You are right, this is inconsistent. We do plan to work on this, and to improve android keystore integration API. But it is not a priority and will take some time. Sorry.
